src/Uno.Extensions.Navigation.Generators/Uno.Extensions.Navigation.Generators.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<IsPackable>false</IsPackable>
+
+		<!--
+			Defines that this project output is a tool of the given package ID,
+			so nuget override will copy it in the 'tools' directory of that package instead of the 'lib' of the current package folder.
+		-->
+		<ToolOfPackage>Uno.Extensions.Navigation</ToolOfPackage>
+
+		<NoWarn>$(NoWarn);RS2008</NoWarn> <!-- RS2008: Enable analyzer release tracking for the analyzer project -->
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Uno.Roslyn" PrivateAssets="All" />
```

Comment:
btw, which methods are we calling from Uno.Roslyn?

Reply:
It's probably not calling any. The references were copied from one of the other generators in the solution, so perhaps we can remove that reference.

Reply:
we're calling some of the helper methods so the reference needs to stay but I'm guessing it's optimized out of the final library

Reply:
Oh it's because it's a source package. Uno.Roslyn doesn't actually have any dlls, just plain `.cs` files in the package that gets into consuming projects.